src/worker.jl

```diff
@@ -438,14 +438,15 @@ end
 Send the results of `keys` back over the stream they were requested on.
 
 # Returns
-- `Dict{Vector{UInt8}, Vector{UInt8}}`: dictionary mapping keys to their serialized data for
+- `Dict{String, Vector{UInt8}}`: dictionary mapping keys to their serialized data for
     communication
 """
 function get_data(worker::Worker; keys::Array=String[], who::String="")
     keys = Vector{String}(keys)
     debug(logger, "\"get_data\": ($keys: \"$who\")")
-    return Dict{Vector{UInt8}, Vector{UInt8}}(
-        Vector{UInt8}(k) =>
+
+    return Dict{String, Vector{UInt8}}(
+        k =>
```

Comment:
Put the pair on one line and if that line is too long put the `for` loop part on the next line, indented